### DISCUSSION TITLE
Der BM ist aber noch deaktiviert bei den FIT Box ist das richtig so?

### DISCUSSION THREAD

**Discussion by freetz-ng-mod** [Freetz-NG/freetz-ng#517] [Allgemein] [answered]
https://github.com/Freetz-NG/freetz-ng/blob/b62c9b062db2f5991527428243074bc279f4faef/config/ui/patches.in#L1294

**Comment by fda77** [comment]
Kein Plan, hab den noch nie genutzt. Hattest nicht du den getestet?

**Comment by freetz-ng-mod** [comment]
Also ich habe freetz nun wieder auf den Original-Stand bei mir gebracht und wollte es testen. Aber Kein BM zusehen ;-).
wenn ich das lösche !FREETZ_AVM_HAS_FWLAYOUT_6  sehe ich ihn ja wieder

**Comment by fda77** [comment]
Wenn du auf die Datei gehst die du nicht verlinkt hast und dann oben "blame" anklickst siehst du dass diese Zeile 1294 seit 21.2.2022 nicht verändert wurde

**Reply by fda77** [reply]
In diesem Commit wurden auch 2 Issues in unterschiedlichen Repos verlinkt.

**Reply by freetz-ng-mod** [reply]
Also bleibt der BM in Freetz aus oder was

**Comment by fda77** [comment]
Solang deinen Aussagen auf Glauben basieren scheint es so

**Comment by freetz-ng-mod** [comment]
Der BM lauft doch auf allen 4 Boxen bei mir. Aber durch dein !FREETZ_AVM_HAS_FWLAYOUT_6 ist er halt für alle deaktiviert.

Und das sagte ich jetzt hier schon einmal und auch im BW Thread von Peter

**Comment by fda77** [comment, marked as answer]
Ich verfolge keine externen Repos.
Wenn sich am fit-support des bm was geändert hat wird es wohl eine neue stable Version davon geben (darauf hab ich eine Benachrichtigung) und eine Eintrag in der Changelog in der das steht. Ich werd keine 100 Seiten Fliesstext durchacken um das herauszufinden!
Wie du dir selbst helfen kannst:
 - https://freetz-ng.github.io/freetz-ng/SUPPORT.html#pull-request
 - https://freetz-ng.github.io/freetz-ng/SUPPORT.html#package-bump
